pygenie/jobs/core.py

```diff
@@ -203,6 +204,15 @@ def _add_dependency(self, dep):
         if dep not in self._dependencies:
             self._dependencies.append(dep)
 
+    def _add_config(self, config):
```

Comment:
Does this automagically work for both a single config (i.e. argument type string) and a collection of them (argument type list)?